docs/security/cosmosdb-jupyter-notebook-2021-08-12.md

```diff
@@ -0,0 +1,67 @@
+
+# Content
+This document describes the mitigation of a security issue described [here](https://msrc-blog.microsoft.com/2021/08/27/update-on-vulnerability-in-the-azure-cosmos-db-jupyter-notebook-feature/).
```

Comment:
if there is a CVE no better refer to that, it is common sense

Reply:
i am not aware that there is a CVE on this vulnerability.